docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,769 @@
+# Working with CHIP Tool
+
+The CHIP Tool (`chip-tool`) is a tool that allows to commission a Matter device
+into the network and to communicate with it using the Zigbee Cluster Library
+(ZCL) messages.
+
+<hr>
+
+-   [Source files](#source)
+-   [Building chip-tool](#building)
+-   [Using chip-tool for Matter accessory testing](#using)
+-   [Supported commands and options](#commands)
+
+<hr>
+
+<a name="source"></a>
+
+## Source files
+
+You can find source files of the CHIP Tool in the `examples/chip-tool`
+directory.
+
+<hr>
+
+<a name="building"></a>
+
+## Building
+
+Before you can use the `chip-tool`, you must compile it from the source on Linux
+(amd64 / aarch64) or macOS.
+
+> To ensure compatibility, build the `chip-tool` and the Matter device from the
+> same revision of the connectedhomeip repository.
+
+To build and run the `chip-tool`:
+
+1. Install all necessary packages and prepare the build system. For more
+   details, see the [Building Matter](BUILDING.md) documentation:
+
+    ```
+    sudo apt-get update
+    sudo apt-get upgrade
+
+    sudo apt-get install git gcc g++ python pkg-config libssl-dev libdbus-1-dev libglib2.0-dev libavahi-client-dev ninja-build python3-venv python3-dev python3-pip unzip libgirepository1.0-dev libcairo2-dev bluez
+    ```
+
+    If the `chip-tool` is built on a Raspberry Pi, install additional packages
+    and reboot the device:
+
+    ```
+    sudo apt-get install pi-bluetooth
+    sudo reboot
+    ```
+
+2. Clone the Project CHIP repository:
+
+    ```
+    git clone https://github.com/project-chip/connectedhomeip.git
+    ```
+
+3. Enter the `connectedhomeip` directory:
+
+    ```
+    cd connectedhomeip
+    ```
+
+4. Initialize the git submodules:
+
+    ```
+    git submodule update --init
+    ```
+
+5. Run the following command:
+
+    ```
+    ./scripts/examples/gn_build_example.sh examples/chip-tool BUILD_PATH
+    ```
+
+    `BUILD_PATH` specifies where the target binaries shall to be placed.
+
+6. To run the tool execute the following command:
+
+    ```
+    ./BUILD_PATH/chip-tool
+    ```
+
+    As a result, a list of available clusters will show up. Each cluster can be
```

Comment:
That's totally true. The thing is that the `chip-tool` itself treats all commands as clusters in all printouts (as you pointed out). So I decided to keep the chip-tool denomination to avoid confusion, but ok, it's not really good. I will fix it.